From ea11f8615ffd367603fb180db6cd1361d06dbaca Mon Sep 17 00:00:00 2001
From: Ben Boeckel <mathstuf@gmail.com>
Date: Wed, 9 Jul 2025 21:34:23 -0400
Subject: [PATCH 2/2] python: support Python 3.14

Fedora Rawhide has it. CI doesn't test it because Github doesn't have
3.14 Python setup yet.
---
 CHANGELOG.rst  | 1 +
 pyproject.toml | 3 ++-
 tox.ini        | 1 +
 3 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 46434d19b..2a39b7e40 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -20,10 +20,11 @@ classifiers = [
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
   "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: 3.14",
   "Topic :: Communications",
   "Topic :: Utilities",
 ]
-requires-python = ">=3.9,<3.14"
+requires-python = ">=3.9,<3.15"
 dependencies = [
     "click>=3.2",
     "click_log>=0.2.0",
diff --git a/tox.ini b/tox.ini
index bc48d85f5..797a3317a 100644
--- a/tox.ini
+++ b/tox.ini
@@ -22,6 +22,7 @@ python =
     3.11: py311
     3.12: py312
     3.13: py313
+    3.14: py314
 
 [testenv:docs]
 allowlist_externals = make
